Dockerfile

```diff
@@ -1,26 +1,31 @@
 # Environment: Jupyter 1.0.0 with IPython 5.0.0 kernel on CentOS7
 FROM centos:7
 
+ENV LC_ALL=en_US.UTF-8
+ENV LANG=en_US.UTF-8
+
 # hadolint ignore=DL3033
 RUN yum install -y epel-release && yum clean all
 
 # hadolint ignore=DL3033
 RUN yum install -y \
       gcc \
-      python-devel \
-      python-pip \
+      python3-devel \
+      python3-pip \
       && yum clean all
 
+RUN ln -fs /usr/bin/python3 /usr/bin/python
```

Comment:
Is this necessary? If we change system Python to use py3 instead of py2, some system utilities may break... E.g. `yum` wouldn't work anymore:

```console
$ docker run -i -t --rm centos:7 /bin/bash
[root@86f896269212 /]# yum install python3
[root@86f896269212 /]# ln -fs /usr/bin/python3 /usr/bin/python
[root@86f896269212 /]# /usr/bin/python -V
Python 3.6.8
[root@86f896269212 /]# yum update
  File "/usr/bin/yum", line 30
    except KeyboardInterrupt, e:
                            ^
SyntaxError: invalid syntax
```

It would be safer to keep both py2 and py3...